changelog/borrow_for_refcounted.dd

```diff
@@ -0,0 +1,54 @@
+Added `SafeRefCounted`, that can be used in `@safe` with `-preview=dip1000`.
+
+`RefCounted` is only available for `@system` code, because of possibility of
+escaping a reference to its payload past the end of its lifetime. We have
```

Comment:
I would use the impersonal "A modified version of "RefCounted" has been added, `std.typecons.SafeRefCounted` [...]".